test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/smoke/KieServerWithBuiltKjarIntegrationTest.java

```diff
@@ -51,7 +52,7 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-@Category(Smoke.class)
+@Category({Smoke.class, OperatorNotSupported.class})
```

Comment:
Why is operator not supported?

Reply:
You asked me to unsupport the method withContainerDeployment in this comment: https://github.com/kiegroup/kie-cloud-tests/pull/595#discussion_r389621436

Reply:
Oh, right, I forgot on that comment.
It is correct.